service/health/searcher/aws.go

```diff
@@ -2,12 +2,43 @@ package searcher
 
 import (
 	"context"
+
+	"github.com/giantswarm/health-service/service/health/key"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // searchAWSCR searches for the cluster config in AWSClusterConfigs resources.
 func (s *Service) searchAWSCR(ctx context.Context, request Request) (*Response, error) {
+
+	awsCR, err := s.g8sClient.ProviderV1alpha1().AWSConfigs("default").Get(request.ClusterID, v1.GetOptions{})
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	var clusterHealth = key.HealthGreen // Optimistic default
```

Comment:
Still `var` is not needed - `clusterHealth := key.HealthGreen` is more concise.

Reply:
clusterHealth has not been declared previously by this point so it is needed here